app/components/primer/beta/button.pcss

```diff
@@ -329,4 +329,19 @@ summary.Button {
   &.Button--large {
     width: var(--primer-control-large-size, 40px);
   }
+
+  /* Increase touch target */
```

Comment:
This increases the touch target for `.Button--iconOnly`. There is some small risk that it will overlap something else, but maybe still ok to enable this by default.

![image](https://user-images.githubusercontent.com/378023/187603860-6c8518bd-007f-496f-b4e0-d8301efa6135.png)

/cc @langermank

Reply:
Yeah, I haven't thoroughly tested this yet. We merged it a bit hastily 😅 we can write some previews to try and simulate how it would behave.